medusa/helper/common.py

```diff
@@ -328,6 +328,8 @@ def try_int(candidate, default_value=0):
     try:
         return int(candidate)
     except (ValueError, TypeError):
+        if isinstance(candidate, (bytes, bytearray)):
+            candidate = candidate.decode()
```

Comment:
Is the candidate always 'utf-8' on Py 3 and `ascii` in Py 2?  Decode's default encoding is different on each platform.